docs/more/development-tips.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Tips for developing Yew applications
+---
+
+:::important contribute
+This document only contains information for adding supporting in IntelliJ IDEA and VSCode.
+Feel free to contribute to add instructions for your editor of choice. 
+:::
+
+## Add a template for creating components
+
+### IntelliJ IDEA
+
+1. Navigate to File | Settings | Editor | Live Templates.  
```

Comment:
See the review comment for the same thing in the VS Code section for more explanation :)
```suggestion
1. Navigate to File > Settings > Editor > Live Templates.  
```

Reply:
Replied in the other review